Output a message to show if a package is built with intltool or
XML::Parser installed.

diff --git a/proc/50-build-exec.sh b/proc/50-build-exec.sh
index c03bda3..6ef1649 100644
--- a/proc/50-build-exec.sh
+++ b/proc/50-build-exec.sh
@@ -13,6 +13,14 @@ if arch_findfile -2 prepare > /dev/null; then
     arch_loadfile_strict -2 prepare
 fi

+if [ -e /usr/bin/intltoolize ]; then
+    abinfo "intltool check: $PKGNAME is built with intltool installed!"
+elif [ -e /usr/lib/perl5/vendor_perl/XML/Parser.pm ]; then
+    abinfo "intltool check: $PKGNAME is built without intltool, but with XML::Parser installed!"
+else
+    abinfo "intltool check: $PKGNAME is built without XML::Parser."
+fi
+
 cd "$SRCDIR"

 if ab_typecheck -f "build_${ABTYPE}_check"; then

